apps/protocol-frontend/src/hooks/useContributionList.ts

```diff
@@ -0,0 +1,60 @@
+import { useState } from 'react';
+import { useUser } from '../contexts/UserContext';
+import { useQuery, useInfiniteQuery } from '@tanstack/react-query';
+import { UIContribution } from '@govrn/ui-types';
+import { ListContributionsQueryVariables } from '@govrn/protocol-client';
+
+export const useContributionList = (args?: ListContributionsQueryVariables) => {
```

Comment:
Wondering if we'd want the unique keys `contributionList` and `infiniteContributionList` since they're both used differently in Dashboard and in the tables.

I mention this in another comment in this file too. It seems recommended to use 2 unique keys when having a regular query and an infinite query on the same data source.